### DISCUSSION TITLE
Do you think can be possible to create a Tuner for GTS4 Mini?

### DISCUSSION THREAD

**Discussion by Cotrox** [zepp-health/zeppos-forum#85] [Q&A]
Do you think can be possible to create a Tuner for GTS4 Mini?

**Comment by juan518munoz** [comment]
Music tuner? for that you'd need access to the device microphone, which is not possible with the current zeppOS versions.

ZeppOS v3 is expected to add this feature.
https://github.com/zepp-health/zeppos-forum/files/10531840/Zepp.OS.v3.0.API.Development.Updates.pdf

**Reply by Cotrox** [reply]
Yeah exactly, it could be amazing
